src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-weight: 400;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product-img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__items-container {
+    color: $color-black;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-grey;
+  }
+
+  &__rate-row,
+  &__price-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
```

Comment:
I think align-items isn't working here/ can you check ?

Reply:
you're right